modules/exploits/windows/local/tokenmagic.rb

```diff
@@ -0,0 +1,228 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/post/windows/powershell'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::Windows::Priv
+  include Msf::Exploit::EXE # Needed for generate_payload_dll
+  include Msf::Post::Windows::FileSystem
+  include Msf::Post::Windows::ReflectiveDLLInjection
+  include Msf::Exploit::FileDropper
+  include Msf::Post::File
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Post::Windows::Powershell
+
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'TokenMagic UAC Bypass',
```

Comment:
Perhaps a more descriptive title.

Reply:
I thought that was a bit bare, changed to `Windows Privilege Escalation via TokenMagic (UAC Bypass)` let me know if it could be improved further.